resources/qml/pages/ConversationsPage.qml

```diff
@@ -75,13 +75,13 @@ Page {
                 }
                 Label {
                     id: jidId;
-                    text: jid;
+                    text: qsTr("jid");
```

Comment:
This is the jabber id which gets replaced by c++ code with the contact jid. Not translateable.